src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,8 +1,49 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.File;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+
 public class WorkWithFile {
 
     public void getStatistic(String fromFileName, String toFileName) {
+        try (BufferedWriter bufferedWriter = new BufferedWriter(new FileWriter(toFileName))) {
+            bufferedWriter.write(readFromFile(fromFileName).toString());
+        } catch (IOException e) {
+            throw new RuntimeException("Can`t write to file" + toFileName, e);
+        }
+    }
+
+    private StringBuilder readFromFile(String fromFileName) {
+        File file = new File(fromFileName);
+        int supplyCounter = 0;
```

Comment:
```suggestion
        int supplySum = 0;
```
counter usually represents number of times something happens, here it's about the sum